boring-sys/patches/boring-pq.patch

```diff
@@ -3981,6 +4388,12 @@ index cafae9d17..074ac5906 100644
 +    struct KYBER768_public_key *out_pub, struct KYBER768_private_key *out_priv,
 +    const uint8_t input[KYBER_GENERATE_KEY_BYTES]);
 +
++// IPDWING_generate_key is a deterministic function that outputs a public and
++// private key based on the given entropy.
```

Comment:
Following boringSSL comment conventions, I think you'd want to say here what values are written to which variables.